src/main/resources/templates/not-implemented.html

```diff
@@ -3,11 +3,12 @@
 <html xmlns:th="http://www.thymeleaf.org" xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
 <head th:include="main.html::head"></head>
 <body>
-<header th:replace="main.html::header"></header>
+<header th:replace="main.html::global-header"></header>
 
 <main id="main" role="main">
     <div th:replace="main.html::phase"></div>
-    <h1>This page has not been implemented</h1>
+    <a href="/">&lt; back</a>
+    <header th:replace="main.html::page-header(title='This page has not been implemented yet')"></header>
```

Comment:
this `header` is inside the `main` but on other pages it's outside. is that a problem?